themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -20,7 +20,16 @@
       <?php else: ?>
         <?php if ($holding['availability'] ?? false): ?>
           <?php /* Begin Available Items (Holds) */ ?>
-           <span class="text-success"><?=$this->transEsc("Available")?><link property="availability" href="http://schema.org/InStock"></span>
+            <?php
+              if ((int)$holding['availability'] === \VuFind\ILS\Logic\ItemStatus::STATUS_UNCERTAIN) {
+                $statusClassEsc = 'text-warning';
+                $status = $holding['status'];
+              } else {
+                $statusClassEsc = 'text-success';
+                $status = is_int($holding['availability']) && !empty($holding['status']) ? $holding['status'] : 'Available';
+              }
+            ?>
+            <span class="<?=$statusClassEsc?>"><?=$this->transEsc($status)?><link property="availability" href="http://schema.org/InStock"></span>
```

Comment:
If we change this above, we also need to change it here.

Reply:
Done.